guests/templates/checklist/taxes_deadline.html

```diff
@@ -1,18 +1,48 @@
-{% if guest.payment %}
-  <tr>
-    <td width="25">{{ macros.checklist_image(guest.taxes.status) }}</td>
-    <td><b><a href="taxes?guest_id={{ guest.id }}">Completed W9</a></b></td>
-    <td><i>Deadline:</i> {{ guest.deadline_from_model('taxes')|datetime_local }}</td>
-  </tr>
-  <tr>
-    <td colspan="3">
-      {% if guest.taxes.status %}
-        You have already uploaded your W9 form, but if you need to make changes you can upload a new one
-        using the link above.
-      {% else %}
-        Use the link above to upload a completed W9 form (a link will be provided to a blank form).
-      {% endif %}
-      <br/></br>
-    </td>
-  </tr>
-{% endif %}
+{% if snippet %}
+  {% if guest.payment %}
+    <tr>
+      <td width="25">{{ macros.checklist_image(guest.taxes_status) }}</td>
+      <td><b><a href="taxes?guest_id={{ guest.id }}">
```

Comment:
Looks like the URL was changed from `w9` to `taxes` at some point, but the associated method in `guests.py` is still `w9`. I am generally ok with changing URLs, as long as we provide aliases for those we've already disseminated.

In this case, we should either stick with `w9`, or change to `taxes` everywhere and add a `cherrypy.expose('w9')` to the method in `guests.py`.

As it stands, clicking this link: https://localhost:4443/uber/guests/taxes?guest_id=XXX yields:
```
Sorry, page not found!

404 Not Found
The path '/uber/guests/taxes' was not found.
```

Reply:
Oh, good catch! I'll change things to `taxes`.